Source/Details/NSArray+Diffing.h

```diff
@@ -35,4 +35,12 @@
  */
 - (void)asdk_diffWithArray:(NSArray *)array insertions:(NSIndexSet **)insertions deletions:(NSIndexSet **)deletions compareBlock:(BOOL (^)(id lhs, id rhs))comparison;
 
+/**
+ * @abstract Compares two arrays, providing the insertion, deletion, and move indexes needed to transform into the target array.
+ * @discussion This compares the equality of each object with `isEqual:`.
+ * This diffing algorithm uses a bottom-up memoized longest common subsequence solution to identify differences.
+ * It runs in O(mn) complexity.
+ * The moves are returned in ascending order of their destination index.
```

Comment:
Usually, the moves are returned in a form that applies either before any changes, after insertions, or after deletions - in other words, the order of insert / delete being applied (if at all) does matter for the move indices being valid.

Could you expand a bit on the comment to describe the exact order of application for insertions, deletions, and moves that results in the correct transformation?

Reply:
Will do - these are returned as simple (previousIndex -> newIndex) moves, which is easy to work with and consistent with what vanilla `[UICollectionView performBatchUpdates:completion:]` wants